### PR TITLE
Feat: support for new exchangeSDK

### PR DESCRIPTION
**Pull request by CremaFR**
### 📝 Description

New version of @ledgerhq/wallet-api-exchange-module to manage both SWAP (changelly) and SWAP_NG (Moonpay)
Created new endpoint to start a swap (to not break Sell and cards that are using start without swap params)
Fixed https://ledgerhq.atlassian.net/browse/LIVE-10906 and https://ledgerhq.atlassian.net/browse/LIVE-10896 by using accounts used on swap to determine apps to install. This check quits the exchange app fixing the retry that did not work. 

### ❓ Context

[- **JIRA or GitHub link**: ](https://ledgerhq.atlassian.net/browse/LIVE-10968)

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - 

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Review by kallen-ledger — APPROVED** [review]

**Review by gre — APPROVED** [review]

**Review by kallen-ledger — APPROVED** [review]

**Review by gre — APPROVED** [review]

File: libs/ledger-live-common/src/hw/actions/startExchange.ts [hunk 1]
```diff
@@ -95,11 +98,38 @@ export const createAction = (
     const [state, setState] = useState(initialState);
     const reduxDeviceFrozen = useFrozenValue(reduxDevice, state.freezeReduxDevice);
 
-    const { exchangeType } = startExchangeRequest;
+    const { exchangeType, provider, exchange } = startExchangeRequest;
+    const requireLatestFirmware = true;
```
Comment (valpinkman) [inline libs/ledger-live-common/src/hw/actions/startExchange.ts:102]: why creating a dependency to `requireLatestFirmware` in your useMemo if it's always going to be `true` ? Is it for readability ?

Comment (CremaFR) [reply libs/ledger-live-common/src/hw/actions/startExchange.ts:102]: Yes mostly, and those changes are made to be a closer 100% match to [initSwap](https://github.com/LedgerHQ/ledger-live/blob/5eb574523c75c06c2f0209b33982b16149e6a375/libs/ledger-live-common/src/hw/actions/initSwap.ts#L151) where it is not a constant. 

I can remove if you think it is better

**Review by ofreyssinet-ledger — APPROVED** [review]

File: libs/ledger-live-common/src/wallet-api/Exchange/server.ts [hunk 2]
```diff
@@ -192,8 +261,8 @@ export const handlers = ({
               exchangeType: ExchangeType[params.exchangeType],
               provider: params.provider,
               transaction: tx,
-              signature: Buffer.from(params.hexSignature, "hex").toString(),
-              binaryPayload: Buffer.from(params.hexBinaryPayload, "hex").toString(),
+              signature: params.hexSignature,
+              binaryPayload: params.hexBinaryPayload,
```
Comment (Justkant) [inline libs/ledger-live-common/src/wallet-api/Exchange/server.ts:265]: Isn't this change a bit breaking ?
And from the variable names it looks correct to go from the hex string to a binary or utf8 string

**Review by Justkant — COMMENTED** [review]
Some questions about some details but looks good for the rest

File: libs/ledger-live-common/src/wallet-api/Exchange/server.ts (hunk 2 above)
Comment (Justkant) [reply libs/ledger-live-common/src/wallet-api/Exchange/server.ts:265]: Ok yeah a rename might good

**Review by Justkant — APPROVED** [review]

**Pull request merged**